pkg/capabilities/cli/cmd/generate_types.go

```diff
@@ -10,13 +10,14 @@ import (
 
 	"github.com/atombender/go-jsonschema/pkg/generator"
 	"github.com/atombender/go-jsonschema/pkg/schemas"
+	"github.com/iancoleman/strcase"
 
 	"github.com/smartcontractkit/chainlink-common/pkg/capabilities"
 )
 
 // CapabilitySchemaFilePattern is used to extract the package name from the file path.
 // This is used as the package name for the generated Go types.
-var CapabilitySchemaFilePattern = regexp.MustCompile(`([^/]+)_(action|trigger|consensus|target)-schema\.json$`)
+var CapabilitySchemaFilePattern = regexp.MustCompile(`([^/]+)_(action|trigger|consensus|target|common)-schema\.json$`)
```

Comment:
What does it mean for a capabilityType to be common? This isn't one of the supported types

Reply:
It's just for the files that share types across the schemas.  There's an example in this PR, I moved signed report out of the datafeeds's schema file so identical and other methods can re-use it (PR for identical will come after this one).